### ISSUE TITLE
ListBox: Role must have accessible name (ARIA issue)

### ISSUE THREAD

**Issue by KubaGorka** [primefaces/primereact#8280]
### Describe the bug

The ListBox component fails accessibility requirements. The element with role="listbox" does not have an accessible name.

This can be observed in the official documentation:
https://primereact.org/listbox/

Example node:
`<ul class="p-listbox-list" role="listbox" aria-multiselectable="false" tabindex="-1" data-pc-section="list">
`

### System Information

```Shell
primereact: 10.9.7
```

### Steps to reproduce the behavior

Go to https://primereact.org/listbox/

Inspect the main <ul> element of the ListBox

Notice that role="listbox" has no accessible name

### Expected behavior

The role="listbox" element should have an accessible name (e.g., via aria-label or aria-labelledby).
